src/Avalonia.Controls/ExitMode.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) The Avalonia Project. All rights reserved.
+// Licensed under the MIT license. See licence.md file in the project root for full license information.
+
+namespace Avalonia
+{
+    public enum ExitMode
```

Comment:
Could you add some XML documentation to this enum?

Reply:
Will add some comments